From cd2f3da20b55189d443562716435cde16082cf57 Mon Sep 17 00:00:00 2001
From: Sai Hari Chandana Kalluri <chandana.kalluri@xilinx.com>
Date: Fri, 14 Jan 2022 12:59:21 -0800
Subject: [PATCH] common_proc.tcl

Signed-off-by: Sai Hari Chandana Kalluri <chandana.kalluri@xilinx.com>
---
 device_tree/data/common_proc.tcl | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/device_tree/data/common_proc.tcl b/device_tree/data/common_proc.tcl
index cb69005..227ee42 100644
--- a/device_tree/data/common_proc.tcl
+++ b/device_tree/data/common_proc.tcl
@@ -275,6 +275,8 @@ proc get_intr_id {drv_handle intr_port_name} {
 			} elseif {[string match "[get_property IP_NAME $intc]" "axi_intc"] } {
 				set cur_intr_info "$intr_id $intr_type"
 			}
+                        } elseif {[string match "[get_property IP_NAME $intc]" "axi_intc"] } {
+                               set cur_intr_info "$intr_id $intr_type"
 		} elseif {[string match -nocase $intc "psu_acpu_gic"]|| [string match -nocase [get_property IP_NAME $intc] "psv_acpu_gic"]} {
 		    set cur_intr_info "0 $intr_id $intr_type"
 		} else {
